src/react/projects/spark-core-react/src/SprkCard/SprkCard.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import warning from 'warning';
+import SprkCardHighlightedHeader from './components/SprkCardHighlightedHeader/SprkCardHighlightedHeader';
+import SprkCardTeaser from './components/SprkCardTeaser/SprkCardTeaser';
+
+const SprkCard = props => {
+  const {
+    additionalClasses,
+    children,
+    highlightedHeaderConfig,
+    idString,
+    standout,
+    teaserConfig,
+  } = props;
+
+  const highlightedHeaderConfigCount = Object.keys(highlightedHeaderConfig)
+    .length;
+  const teaserConfigCount = Object.keys(teaserConfig).length;
+  const onlyHasHighlightedHeader =
+    highlightedHeaderConfigCount > 0 && teaserConfigCount <= 2;
+  const onlyHasTeaser =
+    teaserConfigCount > 2 && highlightedHeaderConfigCount <= 0;
+  const hasNoVariant =
+    highlightedHeaderConfigCount <= 0 && teaserConfigCount <= 2;
```

Comment:
Are these checks correct? If I do this...

`highlightedHeaderConfigCount = 1;
teaserConfigCount = 1;`

it would evaluate `onlyHasHighlightedHeader` to `true`. Is that right?

Reply:
Talked with you about this in person, this is resolved.